docs/cudf/source/cudf_pandas/faq.md

```diff
@@ -181,6 +181,32 @@ There are a few known limitations that you should be aware of:
    ```
 - `cudf.pandas` (and cuDF in general) is only compatible with pandas 2. Version
   24.02 of cudf was the last to support pandas 1.5.x.
+- In order for `cudf.pandas` to produce a proxy array that ducktypes as a NumPy
+  array, we create a proxy type that actually subclasses `numpy.ndarray`. We can
+  verify this with an isinstance check.
+
+  ```python
+  %load_ext cudf.pandas
```

Comment:
Minor (and throughout this doc), if there's a ` ```ipython ` block we can use here, we should prefer that since the `%` magic syntax doesn't work in the default `python` repl.

Reply:
Will do.